### PR TITLE
Update finder-test.sh

### PR DESCRIPTION
**Pull request by lnxblog**
Modify WRITEDIR to ensure writer.sh creates missing directories

### CODE REVIEW

**Comment by dwalkes** [comment]
@lnxblog I think this could have a similar problem in that the student could create once by hand and then all future runs would succeed.
The way I was suggesting to change this is to modify https://github.com/cu-ecen-aeld/assignment-autotest/blob/3d8e2463334d7cd004a8f558a0846969689a3ba8/test/assignment1/assignment-test.sh#L21 to add logic which verifies `writer.sh` completes with success (and actually writes the file with the correct content) when we specify an arbitrary path like `/tmp/aeld-data/$RANDOM/testfile.txt`

File: finder-app/finder-test.sh [hunk 1]
```diff
@@ -22,6 +22,7 @@ then
 else
 	NUMFILES=$1
 	WRITESTR=$2
+	WRITEDIR=/tmp/$3
```
Comment (dwalkes) [inline finder-app/finder-test.sh:25]: @lnxblog using `/tmp/aeld-data` or similar might be good here so students don't have a bunch of random directories in `/tmp` to cleanup.  Also automatically deleting after a successful run would be good.

Comment (lnxblog) [reply finder-app/finder-test.sh:25]: @dwalkes The third argument is provided by assignment-test.sh for creating directory with random name. https://github.com/cu-ecen-aeld/assignment-autotest/pull/21
Any suggestions?

Comment (dwalkes) [reply finder-app/finder-test.sh:25]: @lnxblog I meant to use `/tmp/aeld-data/$3` so `rm -rf /tmp/aeld-data` will clean up all including any random directories

**Comment by dwalkes** [comment]
Looking great @lnxblog , thanks!
Next steps:
1) Rebase this branch into a single commit.
2) Add a pull request at https://github.com/cu-ecen-aeld/assignments-complete-private/tree/assignment-1-complete with a merge of this branch content and showing a passing unit test.
3) Add a pull request at https://github.com/cu-ecen-aeld/assignments-complete-private/tree/assignment-2-complete merging this branch as well, prove the assignment 2 test still completes successfully as well.

**Pull request merged**